docs/source/specs/openapi.yml

```diff
@@ -309,14 +305,214 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/Error"
+                $ref: '#/components/schemas/Error'
         '500':
           description: Unexpected Error
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/Error"
-  /api/v1/users/{uuid}/reset-password/:
+                $ref: '#/components/schemas/Error'
+  '/api/v1/users/{uuid}/preferences/':
+    post:
+      summary: Create a user preference
+      operationId: createUserPreference
+      tags:
+        - UserPreference
+      parameters:
+        - in: header
+          name: Authorization
+          description: e.g. Token 45138a913da44ab89532bab0352ef84b
+          schema:
+            type: string
+          required: true
+        - name: uuid
+          in: path
+          description: ID of user to get
+          required: true
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        description: Preference to add to a Preference
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UserPreferenceIn'
+      responses:
+        '201':
+          description: An object describing the user
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserPreferenceOut'
+        '500':
+          description: Unexpected Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    get:
+      summary: List the user's preferences
+      operationId: listUserPreferences
+      tags:
+        - UserPreference
+      parameters:
+        - in: header
+          name: Authorization
+          description: e.g. Token 45138a913da44ab89532bab0352ef84b
+          schema:
+            type: string
+          required: true
+        - name: uuid
+          in: path
+          description: ID of user to get
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: A paginated list of preference objects
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserPreferencePagination'
+        '500':
+          description: Unexpected Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  '/api/v1/users/{user_uuid}/preferences/{pref_uuid}':
```

Comment:
Can you end this with a slash? Django can be particular so I want to have the docs always have the ending `/`.

`'/api/v1/users/{user_uuid}/preferences/{pref_uuid}/':`